packages/babel-helpers/src/helpers/get.ts

```diff
@@ -0,0 +1,38 @@
+/* @minVersion 7.0.0-beta.0 */
+
+import _superPropBase from "./superPropBase.ts";
+
+// https://tc39.es/ecma262/multipage/reflection.html#sec-reflect.get
+//
+//  28.1ak.5 Reflect.get ( target, propertyKey [ , receiver ] )
+
+type Object = {
+  __proto__?: any;
+} & { [key: string]: unknown };
+
```

Comment:
I re-used this bit in some places rather than extending `object` since `__proto__` isn't "official", should I remove the usage of `__proto__` instead?

Reply:
That's not how to properly type `object` IMHO. I'd rather stick with `(as any).__proto__` than declaring a type with a name `Object`.